src/index_runner/main.py

```diff
@@ -35,13 +37,28 @@ def main():
     # All worker groups to send kafka messages to
     receivers = [es_indexers, releng_importers]
 
+    # used to check update every minute
+    last_updated_minute = int(time.time()/60)
+    _CONFIG_TAG = _query_for_config_tag()
+
     # Initialize and run the Kafka consumer
     consumer = _set_consumer()
 
     while True:
         msg = consumer.poll(timeout=0.5)
         if msg is None:
             continue
+        curr_min = int(time.time()/60)
+        if curr_min > last_updated_minute:
+            config_tag = _query_for_config_tag()
+            if config_tag != _CONFIG_TAG:
```

Comment:
We should check if config_tag is not None here. We don't want to take any action if we fail to fetch the release tag